packages/runtime-core/src/hydration.ts

```diff
@@ -41,7 +41,7 @@ let hasMismatch = false
 const isSVGContainer = (container: Element) =>
   /svg/.test(container.namespaceURI!) && container.tagName !== 'foreignObject'
 
-const isComment = (node: Node): node is Comment =>
+export const isComment = (node: Node): node is Comment =>
```

Comment:
This doesn't feel right to me.
Is there any way to not expose another export to fix this issue?

Reply:
OK，it should use relative import.